lib/PackingPeanut.rb

```diff
@@ -3,6 +3,8 @@
   raise "The PackingPeanut gem must be required within a RubyMotion project Rakefile."
 end
 
+require "moran"
```

Comment:
One niggling issue: it's a bit of a bummer that this has to be outside of the `Motion::Project::App` block, where we could take advantage of knowing what platform we're actually on, but it didn't work when I tried to do that. This complicates development of cross-platform gems - we might need to open a separate support issue for ideas on how to address that.